mobile/src/main/java/org/openhab/habdroid/ui/activity/ContentController.java

```diff
@@ -232,16 +235,31 @@ public void indicateNoNetwork(CharSequence message) {
         mActivity.updateTitle();
     }
 
+    /**
+     * Indicate to the user that no network connectivity is present and Wi-Fi is disabled.
+     *
+     * @param message Error message to show
+     */
+    public void indicateEnableWifiNetwork(CharSequence message) {
```

Comment:
Couldn't this be solved by an additional parameter to `indicateNoNetwork` (e.g. `shouldSuggestEnablingWifi`)